docs/apis/http/tus_upload.md

```diff
@@ -13,6 +13,35 @@ In situations where file uploads might be interrupted due to network issues, bro
 tus ensures that uploads can be resumed from the point of failure without losing data.
 This documentation shows some basic examples, refer [tus official site](https://tus.io/protocols/resumable-upload) for more details.
 
+## Supported tus Features
+
+The backend announces certain tus features to clients. WebDAV responses come with tus HTTP headers for the offical tus features, and additional, ownCloud specific features are announced via the capabilities endpoint (e.g. `https://localhost:9200/ocs/v1.php/cloud/capabilities?format=json`).
+
+```json
+{
+  "ocs": {
+    "data": {
+      "capabilities": {
+        "files": {
+          "tus_support": {
+              "version": "1.0.0",
+              "resumable": "1.0.0",
+              "extension": "creation,creation-with-upload",
+              "max_chunk_size": 10000000,
+              "http_method_override": ""
+            }
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+| Parameter      | Environment Variable           | Default Value | Description                                                         |
+| -------------- | ------------------------------ | ------------- | ------------------------------------------------------------------- |
+| max_chunk_size | FRONTEND_UPLOAD_MAX_CHUNK_SIZE | 10000000      | Announces the max chunk sizes in bytes for uploads via the clients. |
```

Comment:
Where is the environement variable relevant? To set the default for the capabilities, or at the desktop client for example?